jwst/extract_2d/grisms.py

```diff
@@ -208,8 +206,10 @@ def extract_tso_object(input_model,
         ymax = int(ymax)
 
         log.info("WCS made explicit for order: {}".format(order))
-        log.info("Spectral trace extents: (xmin:{}, ymin:{}), (xmax:{}, ymax:{})".format(xmin, ymin, xmax, ymax))
-        log.info("Extraction limits: (xmin:{}, ymin:{}), (xmax:{}, ymax:{})".format(xmin_ext, ymin, xmax_ext, ymax))
+        log.info("Spectral trace extents: (xmin: {}, ymin: {}), "
+                 "(xmax: {}, ymax: {})".format(xmin, ymin, xmax, ymax))
+        log.info("Extraction limits: (xmin: {}, ymin: {}), "
+                 "(xmax: {}, ymax: {})".format(xmin_ext, ymin, xmax, ymax))
```

Comment:
I realize this is the same code as before, just reformatted, but why does this message use `xmin_ext`, while all the other items being printed are just plain `ymin`, `xmax`, `ymax`?

Reply:
The team has decided they want to cover the entire detector in the `x` direction, that's why it's hard coded to `xmin_ext=0`.

Reply:
I see that in the old code the extraction limits (line 212) arguments include `xmin_ext` and `xmax_ext`, while the new code uses `xmin_ext` and `xmax`. Should that instead be `xmax_ext`?